CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.9.14 / 2018-11-26
```

Comment:
We usually just place `xxx` into the CHANGELOG instead of a version number and a date for pull requests, so we don't have merge conflicts later, but you couldn't have known that so no worries. Actually, thanks for being so thorough and updating the CHANGELOG.